components/Intro.vue

```diff
@@ -96,13 +97,14 @@ export default {
       }
     }
   },
-
   data: function () {
     return {
-      parallaxDelay: 300,
-      textParallaxStrength: -200,
-      foregroundParallaxStrength: -160,
-      backgroundParallaxStrength: -75
+      parallaxDelay: 150
+    }
+  },
+  computed: {
+    textParallaxStrength() {
+      return screen.width > 768 ? -350 : -150
```

Comment:
Mightve branched off of the wrong branch.. but should be ok

Reply:
ey i did this too lmao